mvc/controllers/areas.rst

```diff
@@ -1,8 +1,132 @@
-.. include:: /../common/stub-topic.txt
+Areas
+=====
+By `Tom Archer`
 
-|stub-icon| Areas
-===========================================
+MVC uses a folder hierarchy pattern that defines the relationship between your controllers, views, and actions. You can see this with the default wizard-generated ASP.NET MVC app where you start with a ``Controllers`` folder containing several controllers (``Account``, ``Home``, and ``Manage``). If you open the ``HomeController.cs`` file, you'll note that the ``HomeController`` controller has three actions - ``Index``, ``About``, and ``Contact`` - that return a view object without qualifying which view to return. (A fourth action - ``Error`` - is present, but it explicitly names the location of the view to return.) So, how does MVC know which view to return when its location is not explicitly stated?
```

Comment:
we don't have editors so lets do it right - drop the redundant ``HomeController`` controller  or write it as Home controller.